Source/WebCore/accessibility/isolatedtree/AXIsolatedObject.cpp

```diff
@@ -446,6 +446,86 @@ void AXIsolatedObject::setObjectVectorProperty(AXPropertyName propertyName, cons
 
 void AXIsolatedObject::setProperty(AXPropertyName propertyName, AXPropertyValueVariant&& value)
 {
+    switch (propertyName) {
+    case AXPropertyName::CanSetFocusAttribute:
```

Comment:
Maybe this is an unnecessary micro-optimization. But I wonder if we should gate this switch statement with tons of branches on whether `value` is actually a `bool`? That way every non-bool property doesn't do all of this checking.